test/monitoring/components_alerts_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws packet
+// +build poste2e
+
+package monitoring
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+type AlertTestCase struct {
+	Name      string
+	platforms []testutil.Platform
+	Alerts    []string
+}
+
+func testComponentAlerts(t *testing.T, v1api v1.API) {
+	alertTestCases := []AlertTestCase{
+		{
+			Name:      "metallb-rules",
+			platforms: []testutil.Platform{testutil.PlatformPacket},
+			Alerts: []string{
+				"MetalLBNoBGPSession", "MetalLBConfigStale", "MetalLBControllerPodsAvailability",
+				"MetalLBSpeakerPodsAvailability",
+			},
+		},
+	}
+
+	var err error
+
+	const (
+		waitTime = 30
+		retries  = 20
+	)
+
+	for i := 0; i < retries; i++ {
```

Comment:
If you use `t.Run()` and put retry loop inside the test, then `t.Parallel()` can be used and failing tests will be identified to only single case failing.

Reply:
done.